src/app/static/src/app/components/plots/Filter.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+    <multi-select v-if="multiple"
+                  :options="options"
+                  :model-value="selected"
+                  :placeholder="placeholder"
+                  @update:model-value="updateSelection"/>
+    <single-select v-else
+                   :options="options"
+                   :model-value="selected[0]"
+                   :placeholder="placeholder"
+                   @update:model-value="updateSelection"/>
+
+</template>
+
+<script lang="ts">
+import { SingleSelect, MultiSelect } from "@reside-ic/vue-nested-multiselect";
+import {computed, defineComponent, PropType, ref} from 'vue';
+import i18next from "i18next";
+import {useStore} from "vuex";
+import {RootState} from "../../root";
+import {FilterOption} from "../../generated";
+
+export default defineComponent({
+    props: {
+        filterId: String,
+        multiple: Boolean,
+        selectedOptions: Object as PropType<FilterOption[]>
```

Comment:
multiple and selectionOptions can be pulled out of state in this component, might be cleaner as this component would be in charge of that filter select